discovery/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "librespot-discovery"
+version = "0.1.0"
+authors = ["Paul Lietar <paul@lietar.net>"]
+
+[dependencies.librespot-core]
+path = "../core"
+
+[dependencies]
+base64 = "0.5.0"
+futures = "0.1.8"
+hyper = "0.11.2"
+log = "0.3.5"
+num-bigint = "0.1.35"
+protobuf = "1.1"
+rand = "0.3.13"
+rust-crypto = { git = "https://github.com/awmath/rust-crypto.git", branch = "avx2" }
+serde = "0.9.6"
+serde_derive = "0.9.6"
```

Comment:
Why are ```serde``` and ```serde_derive``` required in this crate?

Reply:
https://github.com/awiouy/librespot/blob/f91a380ea43b11d3caa4622e149c23b8a6f5cc4f/discovery/src/discovery.rs#L29

Reply:
ok, serde required, missed that, but what about serde_derive? I don't see it in use anywhere

Reply:
`#[derive(Clone)]`
https://serde.rs/derive.html
We might (try to) remove serde-json later on

Reply:
ok, fair enough. Not sure why they've kept that seperate...